### PR TITLE
Manage function dependencies

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Add function dependencies tab
- Reorganize code tab

**Related issue(s)**
[Kyma](https://github.com/kyma-project/kyma/pull/6252)

### CODE REVIEW

File: shared/components/CollapsiblePanel/CollapsiblePanel.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+export const CollapsiblePanel = ({
+  children,
+  title,
+  className,
+  actions,
+  isOpenInitially = true,
+}) => {
+  const [isOpen, setIsOpen] = useState(isOpenInitially);
+
+  const swtichOpen = e => {
```
Comment (y-kkamil) [inline shared/components/CollapsiblePanel/CollapsiblePanel.js]: typo: `switchOpen`
```suggestion
  const switchOpen = e => {
```

File: shared/components/CollapsiblePanel/CollapsiblePanel.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    e.stopPropagation();
+    if (e.target === e.currentTarget) {
+      setIsOpen(!isOpen);
+    }
+  };
+
+  return (
+    <Panel className={classNames('collapsible-panel', className)}>
+      <Panel.Header onClick={swtichOpen}>
```
Comment (y-kkamil) [inline shared/components/CollapsiblePanel/CollapsiblePanel.js]: typo: `switchOpen`
```suggestion
      <Panel.Header onClick={switchOpen}>
```

File: shared/components/CollapsiblePanel/CollapsiblePanel.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        <Panel.Head title={title} />
+        <Panel.Actions>
+          {actions}
+          <Button
+            glyph={isOpen ? 'navigation-up-arrow' : 'navigation-down-arrow'}
+            option="light"
+            onClick={swtichOpen}
```
Comment (y-kkamil) [inline shared/components/CollapsiblePanel/CollapsiblePanel.js]: typo: `switchOpen`
```suggestion
            onClick={switchOpen}
```

File: core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/LambdaCode.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { Panel } from 'fundamental-react';
 import { ControlledEditor } from '@monaco-editor/react';
 
-const Code = ({ lambdaCode, setLambdaCode }) => {
+const LambdaCode = ({ code, setCode }) => {
```
[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/LambdaCode.js]: _Trifles_, man.

**Pull request merged**